components/stepper/steps/indicators.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Indicators
+page_title: Indicators
+description: Indicators of the Stepper for Blazor.
+slug: stepper-indicators
+tags: telerik,blazor,stepper,indicators
+published: True
+position: 1
+---
+
+
+# Stepper Indicators
+
+This article explains the configuration of the content that will be rendered in the step indicators. Each step could contain text or icon.
+
+>caption In this article:
+
+* [Indicator Text](#indicator-text)
+* [Indicator Icon](#indicator-icon)
+
+## Indicator Text
+
+Stepper component allows you to use text for its step indicators. You can define the desired `string` for each step through the `Text` parameter the `StepperStep` exposes.
+
+>caption Stepper component with a text indicators. The result form the snippet below.
+
+![Text Indicators](images/text-indicators-example.png)
+
+````CSHTML
+@* Stepper with text indicators *@
+
+<div style="width:500px">
+    <TelerikStepper>
+        <StepperSteps>
+            <StepperStep Text="1"></StepperStep>
+            <StepperStep Text="2"></StepperStep>
+            <StepperStep Text="3"></StepperStep>
+        </StepperSteps>
+    </TelerikStepper>
+</div>
+````
+
+## Indicator Icon
+
+Stepper component allows you to use font icon, sprite or an image for its step indicators. You can define the desired visual content through the following parameters of the `StepperStep`:
+
+* `Icon` - defines the name of the desired Telerik font icon.
+* `IconClass` - defines the CSS class of a desired third party font-icon.
+* `SpriteClass` - defines the classes from your site's stylesheet that produce the desired appearance and background positions for your sprites.
+* `ImageUrl` - defines the `url` of the desired raster image.
+
+More details as well as a list of the available Telerik font icons you can find in the [Built-in Icons article]({%slug general-information/font-icons%}).
+
+![Icon Indicators](images/icon-indicators-example.png)
+
+````CSHTML
+@* Stepper with icon indicators *@
```

Comment:
Use the same order of properties in each step for easier reading.

Reply:
Done